sources/packages/forms/src/form-definitions/sfaa2023-24.json

```diff
@@ -1983,7 +1984,7 @@
                       "refreshOnChange": false,
                       "customClass": "banner-info",
                       "key": "html12",
-                      "customConditional": "show = data.programnotListed || data.offeringnotListed;",
+                      "customConditional": "show = data.myProgramNotListed.programnotListed || data.myStudyPeriodIsntListed.offeringnotListed;",
```

Comment:
What is this change about ?

Reply:
Do we need a null check here ? `data.myProgramNotListed?.programnotListed`

Reply:
![image](https://github.com/bcgov/SIMS/assets/77353155/1fd7b4a9-b4b4-4e30-862b-789ba256acc4)